scripts/joinmarket-qt.py

```diff
@@ -506,12 +477,44 @@ def initUI(self):
 
         donateLayout = self.getDonateLayout()
         innerTopLayout.addLayout(donateLayout, 0, 0, 1, 2)
-        self.widgets = getSettingsWidgets()
```

Comment:
Looks like this is the cause of the stack trace, so I guess a fairly trivial fix (refer to the object itself, `SpendTab.widgets` no longer exists ... I see you already made this edit elsewhere, so I guess just an oversight.)